controller/stats_reporter.go

```diff
@@ -99,49 +99,49 @@ func init() {
 		ItemsInList: stats.Float64(
 			"reflector_items_in_list",
 			"How many items an API list returns to the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		// TODO(mattmoor): This is not in the latest version, so it will
 		// be removed in a future version.
 		ItemsInMatch: stats.Float64(
 			"reflector_items_in_match",
 			"",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		ItemsInWatch: stats.Float64(
 			"reflector_items_in_watch",
 			"How many items an API watch returns to the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		LastResourceVersion: stats.Float64(
 			"reflector_last_resource_version",
 			"Last resource version seen for the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		ListDuration: stats.Float64(
 			"reflector_list_duration_seconds",
 			"How long an API list takes to return and decode for the reflectors",
-			stats.UnitNone,
+			stats.UnitSeconds,
```

Comment:
To be aligned with the rest of the metrics this needs `seconds` as unit, same for `reflector_watch_duration_seconds`.